scene/gui/graph_edit.cpp

```diff
@@ -1453,6 +1453,10 @@ void GraphEdit::_update_connections() {
 		Ref<Gradient> line_gradient = memnew(Gradient);
 
 		float line_width = _get_shader_line_width();
+		if (conn == hovered_connection) {
+			line_width *= 1.0f + ((float)theme_cache.connection_hover_thickness / 100.0f);
```

Comment:
The float cast is not needed here as you divide with a float literal.

Reply:
Thanks for this improvement!